### ISSUE TITLE
Compatibility Issue with v0.6.0

### ISSUE THREAD

**Issue by ApfelUser** [julianlam/nodebb-plugin-archiver#2]
```
1/2 10:20 [17676] - error: Error: Cannot find module './redis'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at Object.<anonymous> (/srv/nodebb/node_modules/nodebb-plugin-archiver/index.js:7:22)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
Error: Cannot find module './redis'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at Object.<anonymous> (/srv/nodebb/node_modules/nodebb-plugin-archiver/index.js:7:22)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
1/2 10:20 [17676] - info: [app] Shutdown (SIGTERM/SIGINT) Initialised.
1/2 10:20 [17676] - info: [app] Database connection closed.

net.js:1236
    throw new Error('Not running');
          ^
Error: Not running
    at Server.close (net.js:1236:11)
    at shutdown (/srv/nodebb/app.js:374:36)
    at process.<anonymous> (/srv/nodebb/app.js:211:7)
    at process.emit (events.js:95:17)
    at process._fatalException (node.js:272:26)
15 restarts in 10 seconds, most likely an error on startup. Halting.
```

**Comment by julianlam** [comment]
Resolved, please try v1.0.0

**Comment by ApfelUser** [comment]
Yeah, it works now :+1: 

But I had to install it over NPM, the control panel didn't offer the update.
